src/pages/songhyunLee/Main/CommentList/Comment/Comment.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+class Comment extends Component {
+  render() {
+    return (
+      <li className="feeds-comment__reply">
+        <div className="user-feeds__profile--img">
+          {this.props.profile !== '' ? (
+            <img src={this.props.profile} alt="피드사진" />
+          ) : (
+            ''
+          )}
```

Comment:
```suggestion
          {!this.props.profile && (
            <img src={this.props.profile} alt="피드사진" />
          )
```
- !== 연산자 불필요해보입니다, 빈 문자열은 falsy 값이기에 not 연산자와 조합해서 빈문자열이 아닐때를 표현 가능합니다.
- true일 경우에만 렌더링되면 된다면 삼항연산자말고 && 연산자를 이용해보세요